GWAS/PCA.ipynb

```diff
@@ -976,6 +976,11 @@
     "    # Load required libraries\n",
     "    library(flashpcaR)\n",
     "    library(dplyr)\n",
+    "    pops = c(${paths(pops):r,})\n",
+    "    pops = str_replace(pops, \"AA\", \"African\")\n",
```

Comment:
This is supposed to be a generic pipeline intended to use on any data-set. Let's stick to the original label. If you want to change the label you need to do that in the phenotype file.